samples/highcharts/demo/vertical-sankey/demo.js

```diff
@@ -0,0 +1,300 @@
+                return name.toUpperCase();
+            },
+            style: {
+                textOutline: 'none',
+                color: '#4a4a4a'
+            }
+        },
+        borderRadius: 0,
+        nodeWidth: 50,
+        linkOpacity: 1,
+        linkColorMode: 'gradient',
+        nodes: [{
+            id: 'Total annual 2010 U.S. energy consumption',
+            color: 'rgba(76, 175, 254,0.8)'
+        }, {
+            id: 'All non-water related energy consumption',
+            color: 'rgba(199, 113, 243, 0.8)'
+        }, {
+            id: 'Water-related energy consumption',
+            color: 'rgba(83, 82, 190, 0.8)'
+        }, {
+            id: 'Non-boiler water-related applications',
+            offsetHorizontal: '200%',
+            color: 'rgba(199, 113, 243, 0.8)',
+            dataLabels: {
+                enabled: false
+            }
+        }, {
+            id: 'Steam boilers',
+            offsetHorizontal: '60%',
+            color: 'rgba(243, 172, 165, 0.8)'
+        }, {
+            id: 'Other (2)',
+            color: 'rgba(112, 138, 185, 0.8)',
+            dataLabels: {
+                rotation: -90
+            }
+        }, {
+            id: 'Non-electric water heating',
+            color: 'rgba(108, 221, 202, 0.8)',
+            dataLabels: {
+                rotation: -85,
+                x: -5,
+                y: 60
+            }
+        }, {
+            id: 'Steam-driven power generation',
+            offsetHorizontal: '55%',
+            color: 'rgba(232, 89, 75, 0.8)'
+        }, {
+            id: 'Non-electric steam applications',
+            offsetHorizontal: '200%',
+            color: 'rgba(136, 157, 197, 0.8)'
+        }, {
+            id: 'Other power (non-steam)',
+            column: 4,
+            offsetHorizontal: '700%',
+            color: 'rgba(108, 221, 202, 0.8)'
+        }, {
+            id: 'To direct water services',
+            offsetHorizontal: '320%',
+            color: 'rgba(76, 175, 254, 0.8)',
+            dataLabels: {
+                rotation: -90
+            }
+        }, {
+            id: 'Other electric devices',
+            offsetHorizontal: '60%',
+            color: 'rgba(250, 183, 118, 0.8)'
+        }, {
+            id: 'Space and process heating',
+            offsetHorizontal: '300%',
+            color: 'rgba(83, 82, 190, 0.8)',
+            dataLabels: {
+                rotation: -80
+            }
+        }, {
+            id: 'Direct steam injection in industrial processes',
+            offsetHorizontal: '450%',
+            color: 'rgba(128, 135, 232, 0.8)',
+            dataLabels: {
+                rotation: -80
+            }
+        }, {
+            id: 'Electric water-related applications',
+            offsetHorizontal: '-250%',
+            color: 'rgba(255, 141, 100, 0.8)',
+            dataLabels: {
+                rotation: -90,
+                x: -10,
+                format: '{addBR point.name}'
```

Comment:
We can achieve word-wrap by setting a width to the data label. Which I think is better because we don't have to add a custom template helper, and can achieve what we want with pure configuration.

```
style: {
     width: '90px'
}
```